src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -228,6 +228,11 @@ func (cfg *Config) applyClientDefaults() {
 		}
 	}
 
+	if cfg.Client.ProxiedSites.Cloud == nil || len(cfg.Client.ProxiedSites.Cloud) == 0 {
+		log.Debugf("Loading default cloud proxiedsites")
```

Comment:
I'm now defaulting the list of cloud proxiedsites based on a list kept in the binary itself.